crates/wasmparser/src/binary_reader.rs

```diff
@@ -1712,6 +1712,12 @@ impl<'a> BinaryReader<'a> {
         self.visit_operator(&mut OperatorFactory::new())
     }
 
+    /// Returns whether there is an `end` opcode followed by eof remaining in
+    /// this reader.
+    pub fn is_end_then_eof(&mut self) -> bool {
```

Comment:
Can this be `&self` instead of `&mut self`?

Reply:
Oop, indeed!